waiter/src/waiter/utils.clj

```diff
@@ -472,3 +473,17 @@
   "Parses the request header to determine if debug mode has been enabled."
   [request]
   (boolean (get-in request [:headers "x-waiter-debug"])))
+
+(let [last-modified-format "EEE, dd MMM yyyy HH:mm:ss zzz"]
```

Comment:
@sradack  One issue with this format is that we can get only upto second granularity (instead of millisecond granularity we have in our timestamps). Do we want to make this sacrifice in accuracy?

Reply:
"If sub-second accuracy is important, it may be more appropriate to use entity tags (ETag)."

https://stackoverflow.com/questions/18603154/http-if-modified-since-with-milliseconds